src/app/archive/archive_lib/processor.ml

```diff
@@ -2781,19 +2854,63 @@ module Block = struct
                           (module Conn)
                           fee_transfer `Via_coinbase
                       in
+                      let status, failure_reason =
+                        match status with
+                        | Applied ->
+                            (applied_str, None)
+                        | Failed failures -> (
+                            (* at most two failures in a coinbase transaction First one for the fee transfer and the second for reward transfer*)
+                            match failures with
+                            | [ []; _ ] ->
+                                applied_status
```

Comment:
should this case ever happen? is it an internal error if it does happen?

Reply:
This is the case where the coinbase part of the transaction could fail but the fee transfer succeeds (the coinbase transaction returns as failed) and we are retrieving only the status of the fee tansfer here